Touch Bar Simulator/ToolbarSlider.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ToolbarSlider.swift
+//  Touch Bar Simulator
+//
+//  Created by Wayne Yeh on 2017/3/17.
+//  Copyright © 2017年 Sindre Sorhus. All rights reserved.
+//
+
+import Cocoa
+
+class ToolbarSliderCell: NSSliderCell {
+	override func drawKnob(_ knobRect: NSRect) {
+		struct Knob {
+			static var image = NSImage()
+		}
+		if Knob.image.size == CGSize.zero {
```

Comment:
Because drawKnob() will be call only on init and Knob is dragging
If you use breakpoint, you can find "Knob.image.size == CGSize.zero" only call once